llmstack/datasources/handlers/databases/sql.py

```diff
@@ -68,37 +83,31 @@ def __init__(self, datasource: DataSource):
             )
             self._source_name = self.datasource.name
 
-    @staticmethod
-    def name() -> str:
-        return "MySQL"
+    def name(self) -> str:
+        return self.datasource.type.name
 
-    @staticmethod
-    def slug() -> str:
-        return "mysql"
+    def slug(self) -> str:
+        return self.datasource.type.slug
 
-    @staticmethod
-    def description() -> str:
-        return "Connect to a MySQL database"
+    def description(self) -> str:
+        return f"Connect to a {self.datasource.type.name} database"
```

Comment:
This too needs to be a static method. This can return a generic description about connecting to SQL like database